generators/cucumber/types.d.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2013-2025 the original author or authors from the JHipster project.
+ *
+ * This file is part of the JHipster project, see https://www.jhipster.tech/
+ * for more information.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { Application as JavaApplication } from '../java/types.js';
+export type Application = JavaApplication & {
+  cucumberTests: boolean;
```

Comment:
I don't see a reason for `cucumberTests` inside cucumber generator.
`cucumberTests` should be in SpringBoot since it's used to compose with cucumber generator and our cucumber implementation is spring specific.

Reply:
May we first focus on type, then on removing these properties using proper composeBlueprint instructions? I'm not comfortable doing both at once (and tbh don't know how to do it properly yet)